internal/command/action_test.go

```diff
@@ -240,3 +240,55 @@ testprovider-trailingnewline version 3.2.1
 		So(output, ShouldEqual, expected)
 	})
 }
+
+func TestLocateFileRecurseUp(t *testing.T) {
+	filename := "test.txt"
+	currentDir, _ := os.Getwd()
+	var err error
+
+	Convey("Finds file in current working directory", t, func() {
+		localFilePath := filepath.Join(currentDir, filename)
+		_, err = os.Create(localFilePath)
+		So(err, ShouldBeNil)
+
+		gotPath, err := walkFn(filename, currentDir)
+		So(err, ShouldBeNil)
+
+		So(gotPath, ShouldEqual, localFilePath)
+
+		err = os.Remove(localFilePath)
```

Comment:
This should be in a `defer` block right under creating the file

Reply:
Removed due to introduction of tmpdir.